tests/turtle/syntax/manifest.jsonld

```diff
@@ -73,134 +76,134 @@
     },
     {
       "@id": "trs:turtle-star-bad-4",
       "@type": "TestTurtleNegativeSyntax",
-      "name": "Turtle-star - bad - literal in subject position of embedded triple",
-      "action": "turtle-star-syntax-bad-04.ttl"
+      "action": "turtle-star-syntax-bad-04.ttl",
+      "name": "Turtle-star - bad - literal in subject position of embedded triple"
     },
     {
       "@id": "trs:turtle-star-bad-5",
       "@type": "TestTurtleNegativeSyntax",
-      "name": "Turtle-star - bad - blank node  as predicate in embedded triple",
-      "action": "turtle-star-syntax-bad-05.ttl"
+      "action": "turtle-star-syntax-bad-05.ttl",
+      "name": "Turtle-star - bad - blank node  as predicate in embedded triple"
     },
     {
       "@id": "trs:turtle-star-bad-6",
       "@type": "TestTurtleNegativeSyntax",
-      "name": "Turtle-star - bad - compound blank node expression",
-      "action": "turtle-star-syntax-bad-06.ttl"
+      "action": "turtle-star-syntax-bad-06.ttl",
+      "name": "Turtle-star - bad - compound blank node expression"
     },
     {
       "@id": "trs:turtle-star-bad-7",
       "@type": "TestTurtleNegativeSyntax",
-      "name": "Turtle-star - bad - incomplete embedded triple",
-      "action": "turtle-star-syntax-bad-07.ttl"
+      "action": "turtle-star-syntax-bad-07.ttl",
+      "name": "Turtle-star - bad - incomplete embedded triple"
     },
     {
       "@id": "trs:turtle-star-bad-8",
       "@type": "TestTurtleNegativeSyntax",
-      "name": "Turtle-star - bad - over-long embedded triple",
-      "action": "turtle-star-syntax-bad-08.ttl"
+      "action": "turtle-star-syntax-bad-08.ttl",
+      "name": "Turtle-star - bad - over-long embedded triple"
     },
     {
       "@id": "trs:turtle-star-ann-1",
       "@type": "TestTurtlePositiveSyntax",
-      "name": "Turtle-star - Annotation form",
-      "action": "turtle-star-annotation-1.ttl"
+      "action": "turtle-star-annotation-1.ttl",
+      "name": "Turtle-star - Annotation form"
     },
     {
       "@id": "trs:turtle-star-ann-2",
       "@type": "TestTurtlePositiveSyntax",
-      "name": "Turtle-star - Annotation example",
-      "action": "turtle-star-annotation-2.ttl"
+      "action": "turtle-star-annotation-2.ttl",
+      "name": "Turtle-star - Annotation example"
     },
     {
       "@id": "trs:turtle-star-bad-ann-1",
       "@type": "TestTurtleNegativeSyntax",
-      "name": "Turtle-star - bad - empty annotation",
-      "action": "turtle-star-syntax-bad-ann-1.ttl"
+      "action": "turtle-star-syntax-bad-ann-1.ttl",
+      "name": "Turtle-star - bad - empty annotation"
     },
     {
       "@id": "trs:turtle-star-bad-ann-2",
       "@type": "TestTurtleNegativeSyntax",
-      "name": "Turtle-star - bad - triple as annotation",
-      "action": "turtle-star-syntax-bad-ann-2.ttl"
+      "action": "turtle-star-syntax-bad-ann-2.ttl",
+      "name": "Turtle-star - bad - triple as annotation"
     }
   ]
 }
```

Comment:
Minor: Convenient if we a have finalnewlines consistently.

Reply:
Might need a tweak of the tools to ensure the ordering of JSON-LD object members stays consistent, if that's what you're referring to.